### ISSUE TITLE
fail to recover sample result with provided pretrained checkpoints

### ISSUE THREAD

**Issue by zkshan2002** [yangzhao1230/GraphTextRetrieval#1]
I downloaded the pretrained checkpoints following the instructions. Here's part of my directory

<img width="262" alt="Screenshot from 2022-11-01 17-00-32" src="https://user-images.githubusercontent.com/79029603/199202485-3f6cea01-40f7-48ad-83da-2c2716b53d2a.png">

But when I tried the sample experiment in readme, I got a far worse accuracy and recall@20

<img width="1167" alt="Screenshot from 2022-11-01 17-01-09" src="https://user-images.githubusercontent.com/79029603/199202507-db590b69-8f3e-4c1f-8479-39f65ab7f518.png">

I can't imagine why, since sample experiment is simply doing a zero-shot testing using provided checkpoints without actual training, and that since the experiment ended with a result, these checkpoints should be loaded correctly. I run the code in an environment with updated version of torch libraries, but I believe this isn't the major reason.
So what may be the possible reasons? Could you please double check the checkpoints or the testing data provided? I'll appreciate your help! 

Environment
- os: ubuntu 20.04
- python 3.8.13
- torch 1.12.0+cu113
- torch-geometric 2.1.0.post1
- torch-cluster 1.6.0
- torch-scatter 2.0.9
- torch-sparse 0.6.15
- torch-spline-conv 1.2.1
- transformers 4.23.1

**Comment by yangzhao1230** [comment]
Hi, thanks for your interest in our work.

I try to double check the checkpoint and reproduce the experiment with a new environment, but I don't meet the issue. Note that I just create a new project and git clone this repository so the testing data  goes well. Maybe you can follow the script below to check if your checkpoint is correct.
"""
import torch

pth = 'MoMu-S.ckpt'
pt = torch.load(pth)
dict = pt['state_dict']

print(dict['graph_encoder.gnns.1.mlp.0.weight'])
"""
The expected output is:
tensor([[ 0.2377,  0.0594,  0.0964,  ..., -0.7122, -0.0049,  0.1441],
        [ 0.2942, -0.1938,  0.2227,  ...,  0.2918,  0.1980, -0.6875],
        [-0.2362, -0.2085,  0.0364,  ...,  0.1781, -0.1764, -0.3177],
        ...,
        [-0.0790, -0.0715,  0.2107,  ..., -0.0668, -0.0781, -0.0123],
        [-0.2110, -0.2474,  0.1675,  ...,  0.1561, -0.1512, -0.3383],
        [-0.1518,  0.0600,  0.0066,  ...,  0.0857, -0.2052, -0.5830]],
       device='cuda:0')

**Comment by qkrdmsghk** [comment]
I have the same issue that cannot recover sample result with provided pretrained checkpoints.
And I have checked the provided pretrained checkpoints and the expected output is the same as the author provided.
<img width="493" alt="截屏2022-11-27 18 04 56" src="https://user-images.githubusercontent.com/76090479/204127295-1dba3bed-845e-4746-90a9-558adfc07bcb.png">

But it still report as follows:
<img width="837" alt="截屏2022-11-27 18 07 59" src="https://user-images.githubusercontent.com/76090479/204127379-039b874a-f579-49ee-8419-eba12cc124ff.png">

Could you please double check the checkpoints or the testing data provided or the scripts? 
I'll appreciate your help!

**Comment by yangzhao1230** [comment]
> dict['graph_encoder.gnns.1.mlp.0.weight']

Hi @qkrdmsghk. 
I'm sorry that some bugs related to data processing existed.
Already fixed.  Please pull and run again. 
Thank you for spotting this bug!

**Comment by yangzhao1230** [comment]
Hi @zkshan2002, thank you for spotting this bug!
Bugs fixed.  Please pull and run again. If you have any further issues with this, please feel free to reopen it.
